### PR TITLE
reference/transactions: update pessimistic doc to v3.0

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Apply updates to v3.0

### What is the related PR or file link(s)? 
- https://github.com/pingcap/docs-cn/pull/1826
- https://github.com/pingcap/docs/pull/1589

### Which version does your change affect? 
v3.0, dev (minor wording edits)

### CODE REVIEW

File: v3.0/reference/transactions/transaction-pessimistic.md [hunk 1]
```diff
@@ -30,56 +26,39 @@ TiDB 默认使用乐观事务模式，存在事务提交时因为冲突而失败
...
 - 执行 `set @@tidb_txn_mode = 'pessimistic';`，使这个 session 执行的所有显式事务（即非 autocommit 的事务）都会进入悲观事务模式。
 
-- 在配置文件设置中默认启用悲观事务模式，使除了自动提交的单语句事务之外的所有事务都会进入悲观事务模式。
-
-    ```
-    [pessimistic-txn]
-    enable = true
-    default = true
-    ```
+- 执行 `set @@global.tidb_txn_mode = 'pessimistic';`，使之后整个集群所有新创建的 session 都会进入悲观事务模式执行显示事务。
```
Comment (dcalvin) [inline v3.0/reference/transactions/transaction-pessimistic.md]:
```suggestion
- 执行 `set @@global.tidb_txn_mode = 'pessimistic';`，使之后整个集群所有新创建的 session 都会进入悲观事务模式执行显式事务。
```

Comment (coocood) [reply v3.0/reference/transactions/transaction-pessimistic.md]: This also needs to be fixed for the `dev` version.

Comment (dcalvin) [reply v3.0/reference/transactions/transaction-pessimistic.md]: @juliezhang1112 Please update as suggested above. Thanks.

**Comment by juliezhang1112** [comment]
PTAL @dcalvin @coocood

**Comment by coocood** [comment]
LGTM

**Comment by dcalvin** [comment]
LGTM

**Pull request merged**